fred2/wing_editor.h

```diff
@@ -51,6 +51,7 @@ class wing_editor : public CDialog
 	int		m_waves;
 	int		m_threshold;
 	int		m_formation;
+	CString	m_formation_scale;
```

Comment:
This is nitpicky, but it would be nice looking if this was aligned with surrounding entries :)

Reply:
If tabs are set to 4 spaces, it lines up. :)